staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -27,6 +27,10 @@ const (
 	// More info: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#label-selector-and-annotation-conventions
 	labelPrefix = "batch.kubernetes.io/"
```

Comment:
BTW, this prefix is (already) used for both labels and annotations.

Reply:
Do we need to create a separate prefix for annotation?

Reply:
We can use the same prefix. It's the name of the const that we might want to tweak.